### ISSUE TITLE
REPL does not understand built-in functions

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/ergo#295]
For instance:
```
ergotop$ return flatten([[]]);
Runtime error at 1:7-1:20 'Function 'org.accordproject.ergo.stdlib.flatten' not found during eval'
```

**Comment by jeromesimeon** [comment]
Might be best handled as part of #42, with refactoring of the built-in library (which currently is written in `ErgoNNRC` rather than `ErgoC`).

**Comment by jeromesimeon** [comment]
Fixed in PR #319:
```
Welcome to ERGOTOP version 0.1.2
ergotop$ return flatten([[]]);
Response. []
ergotop$ return flatten([["this"],["should","be"],["flat"]]);
Response. ["this", "should", "be", "flat"]
```

**Comment by jeromesimeon** [comment]
Note, this does not address #255 (built-in functions for dateTime still won't work in the REPL).
```
ergotop$ return now();
Response. null
```
